CHANGES.md

```diff
@@ -6,6 +6,7 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 ### What's new
 
+* ZooKeeper instances on master nodes can now be backed up and restored via a dedicated command line script `dcos-zk` that is shipped with DC/OS.
```

Comment:
Nit - add a new line

https://github.com/dcos/dcos/wiki/CHANGES.md-guidelines#format

Reply:
Haha, I removed the new-line in a separate commit because I noticed the over entries didn't have one :D